htdocs/contact/list.php

```diff
@@ -55,6 +55,10 @@
 $contextpage = GETPOST('contextpage', 'aZ') ?GETPOST('contextpage', 'aZ') : 'contactlist';
 $mode = GETPOST('mode', 'alpha');
 
+if ($contextpage == 'poslist') {
+	$_GET['optioncss'] = 'print';
```

Comment:
The goal is to select Contact instead of Customer in TakePos.
When you have a BtoB shop, you may need it instead of doing the sale to a customer, then go to the invoice, add the contact and regenerate the pdf so that the contact is on the invoice 
![image](https://github.com/Dolibarr/dolibarr/assets/396322/2523defd-84cb-46bf-ba88-e62175613cd6)

Reply:
for the 'print css, i did exactly the same as in societe/list.php 
![image](https://github.com/Dolibarr/dolibarr/assets/396322/7e45a1f1-0de5-4e89-9207-7b355f0be038)